handlers/homu_status/__init__.py

```diff
@@ -2,6 +2,19 @@
 import json
 import re
 
+WPT_RESULT_ARROW = '\xe2\x96\xb6'
+
+def parse_log(log):     # Currently, this just gets the names of the failed tests
```

Comment:
We should use the errorsummary log for this, parse each line as JSON, and extract the filename from the "test" field.